### ISSUE TITLE
Re-use grouped components instead of having duplicates in Storybook

### ISSUE THREAD

**Issue by cezaraugusto** [brave/brave-browser#5180]
After brave/brave-core#2857 we now have feature components and grouped components stored in the same place. We could take advantage of this and re-use components in Storybook and brave-core. 

This would likely change some components' API but future-wise this practice would improve productivity and avoid common copy/paste mistakes.

Setting as p5 for now until I can investigate how doable it is and how fast we can move.

Self-assigning for tracking purposes.

**Comment by petemill** [comment]
@cezaraugusto Do you have an example of what this is referring to?

**Comment by cezaraugusto** [comment]
yes, if you search for "footerInfo.tsx" you will see two components. one for storybook and one set as container. ideally we would have only one
